db/seed.rb

```diff
@@ -0,0 +1,16 @@
+require "bundler/setup"
+
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "connection"
+require_relative "../models/apartment"
+require_relative"../models/tenant"
+
+Apartment.create address ([
```

Comment:
This won't work with `address` between `.create` and the opening parentheses. The `.create` method looks something like this...

```rb
Apartment.create( address: "123 Main St", monthly_rent: 5000 )
```

I like how you're passing in multiple hashes into the create method though!